internal/fusefrontend/fs.go

```diff
@@ -121,12 +120,12 @@ func (fs *FS) Open(path string, flags uint32, context *fuse.Context) (fuseFile n
 	f, err := os.OpenFile(cPath, newFlags, 0)
 	if err != nil {
 		sysErr := err.(*os.PathError).Err
-		if sysErr == syscall.EMFILE {
-			var lim syscall.Rlimit
-			syscall.Getrlimit(syscall.RLIMIT_NOFILE, &lim)
+		if sysErr == unix.EMFILE {
```

Comment:
Hmm, this is a bit ugly. The error is returned by `os.OpenFile`, which uses the syscall package internally. But then we compare against the error definition in the unix package...